docs/src/pages/[platform]/components/authenticator/configuration/hidesignup.default.angular.mdx

```diff
@@ -0,0 +1,3 @@
+```html{1} file=../../../../../../../examples/angular/src/pages/ui/components/authenticator/sign-in-with-email/sign-in-with-email.component.html
```

Comment:
I had to fix the imports for a lot of files, since these relative paths were all wrong after moving the files around. Perhaps we should look at trying to use a different import pattern, for the future. Or move all the code into these files, instead of importing them in. 

That can be a future EE item.

Reply:
Yea these are pretty ugly imports. I do like having them import the example code though so that our integration tests are testing the code shown in the docs.